frontend/packages/dev-console/src/components/pipelines/pipeline-builder/types.ts

```diff
@@ -6,12 +6,25 @@ import {
   PipelineTask,
 } from '../../../utils/pipeline-augment';
 import { PipelineVisualizationTaskItem } from '../../../utils/pipeline-utils';
+import { AddNodeDirection } from '../pipeline-topology/const';
+import { PipelineTaskListNodeModel } from '../pipeline-topology/types';
+import { TaskErrorType, UpdateOperationType } from './const';
+
+export type PipelineBuilderTaskBase = { name: string; runAfter?: string[] };
+
+export type PipelineBuilderListTask = PipelineBuilderTaskBase & {};
```

Comment:
why `& {}` ?
Can it contain any field with any value?

Reply:
I don't think `{}` means anything special... it just means an empty object (no key/values). Which in large is useless... I'll remove it.

It was a hold over from when I didn't have PipelineBuilderTaskBase that contained common items. I just never removed it when I was cleaning it up.